crates/nu-protocol/src/engine/engine_state.rs

```diff
@@ -177,6 +177,18 @@ impl EngineState {
         }
     }
 
+    pub fn interrupt(&self) -> &Interrupt {
```

Comment:
The naming of this function is a little confusing IMO. When I see a _function_ named `interrupt()` called, I read that as a verb and assume that calling the function will interrupt something.

I wonder if something like `get_interrupt()` might be a little more clear (very open to other ideas too)